src/Test.UnitTests.Sarif/ArtifactProviderTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.IO.Compression;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+
+using Xunit;
+
+namespace Test.UnitTests.Sarif
+{
+    public class ArtifactProviderTests
+    {
+
+        [Fact]
+        public void MultithreadedZipArchiveArtifactProvider_RetrieveSizeInBytes()
+        {
+            string entryContents = "test";
+            ZipArchive zip = CreateZipArchive("test.zip", entryContents);
+            var artifactProvider = new MultithreadedZipArchiveArtifactProvider(zip, FileSystem.Instance);
```

Comment:
Curious why we would use `var` here, and the explicit `ZipArchive` type in the previous line.